Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:16.0.0-alpine
+FROM node:16.13.0-alpine
```

```suggestion
FROM node:16-alpine
```
I would not suggest using `16.13.0` because this image won't receive any future updates. `16-alpine` should not cause any breaking changes and will update even beyond node `16.3.x`

---

Yes, that is right, thank you!